.gitignore

```diff
@@ -9,3 +9,4 @@
 .vscode
 **/*.cache
 **/*.egg-info
+**/uploads/
```

Comment:
what does this directory store? :o

Reply:
it stores the files that does not get uploaded because of invalid content type

Reply:
As I am thinking a bit more about this, I think I should use `fs.unlinkSync` in the validator class as well just before we are returning invalid content type error response. That way, the files with invalid content type will not get stored in the uploads directory

Reply:
got it. removing invalid files sounds good to me, and yep should ignore uploads